sys/linux/prctl.txt

```diff
@@ -9,6 +9,12 @@ include <uapi/linux/capability.h>
 include <uapi/linux/securebits.h>
 include <asm/prctl.h>
 
+# Use this variant (which will always return -EINVAL) to explicitly set the 5
+# syscall arguments that will enable to properly infer callArgSizes for all
+# prctl variants.
+
+prctl$0(option const[0], arg2 const[0], arg3 const[0], arg4 const[0], arg5 const[0])
```

Comment:
I think it's better to add 5-th arg to one of existing prctl variants. I.e. a random one with 4 args now.
Or at least mark this as `(disabled)`.
But a good find!

Reply:
OK, it looks like no test actually use a `prctl` variant, only corpus I guess, but I think it is better to let this one (with `(disabled)`) to make sure it will not be removed in a future cleanup.